src/main/java/org/wso2/carbon/connector/amazonsqs/operations/message/SendMessage.java

```diff
@@ -0,0 +1,115 @@
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.carbon.connector.amazonsqs.operations.message;
+
+import org.apache.axiom.om.OMElement;
+import org.apache.commons.lang.StringUtils;
+import org.apache.synapse.MessageContext;
+import org.wso2.carbon.connector.amazonsqs.connection.SqsConnection;
+import org.wso2.carbon.connector.amazonsqs.constants.Constants;
+import org.wso2.carbon.connector.amazonsqs.exception.SqsInvalidConfigurationException;
+import org.wso2.carbon.connector.amazonsqs.utils.Error;
+import org.wso2.carbon.connector.amazonsqs.utils.Utils;
+import org.wso2.carbon.connector.core.AbstractConnector;
+import org.wso2.carbon.connector.core.ConnectException;
+import org.wso2.carbon.connector.core.connection.ConnectionHandler;
+import org.wso2.carbon.connector.core.util.ConnectorUtils;
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest;
+import software.amazon.awssdk.services.sqs.model.SendMessageResponse;
+import software.amazon.awssdk.services.sqs.model.SqsException;
+
+import java.net.MalformedURLException;
+
+/**
+ * Implements send message operation.
+ */
+public class SendMessage extends AbstractConnector {
+
+    @Override
+    public void connect(MessageContext messageContext) throws ConnectException {
+        try {
+            ConnectionHandler handler = ConnectionHandler.getConnectionHandler();
+            SqsConnection sqsConnection = (SqsConnection) handler
+                    .getConnection(Constants.CONNECTOR_NAME, Utils.getConnectionName(messageContext));
+            String queueUrl = Utils.createUrl(messageContext, sqsConnection);
+            String delaySeconds = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    "delaySeconds");
+            String messageAttributes = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    "messageAttributes");
+            String messageBody = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    "messageBody");
+            String messageGroupId = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    "messageGroupId");
+            String messageDeduplicationId = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    "messageDeduplicationId");
+            String messageSystemAttributes = (String) ConnectorUtils.lookupTemplateParamater(messageContext,
+                    "messageSystemAttributes");
+
+            SendMessageRequest.Builder sendMessageBuilder = SendMessageRequest.builder().
+                    queueUrl(queueUrl).messageBody(messageBody);
+
+            if (StringUtils.isNotEmpty(delaySeconds)) {
+                sendMessageBuilder.delaySeconds(Integer.getInteger(delaySeconds));
```

Comment:
If a parser error happens, it's better to throw an `SqsInvalidConfigurationException`

Reply:
Updated with `Integer.valueOf` method. It returns number format exception.